src/app/foundations/chart/chart-demo.component.ts

```diff
@@ -0,0 +1,194 @@
+import {Component} from '@angular/core';
+import {SectionService} from '../../shared/section.service';
+import {BaseDemoComponent} from '../../shared/base-demo.component';
+import { SelectChangeEvent } from '@healthcatalyst/cashmere';
+import {ViewChild, ElementRef} from '@angular/core';
+import * as echarts from 'echarts';
+import * as hcChartLight from '../../../assets/foundations/charts/healthcatalyst-light-echart-v5.x.json';
+import * as hcChartDark from '../../../assets/foundations/charts/healthcatalyst-dark-echart-v5.x.json';
+
+const exampleData = {
+    simple: {
+        legend: undefined,
+        labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+        series: [
+            {
+                name: "Week 1",
+                data: [150, 230, 224, 218, 135, 147, 260],
+            }
+        ],
+    },
+    grouped: {
+        labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+        series: [
+            {
+                name: "Week 1",
+                data: [150, 230, 224, 218, 135, 147, 260],
+            },
+            {
+                name: "Week 2",
+                data: [260, 218, 230, 224, 135, 150, 147],
+            },
+            {
+                name: "Week 3",
+                data: [230, 150, 218, 224, 147, 260, 135],
+            },
+            {
+                name: "Week 4",
+                data: [218, 224, 135, 260, 150, 230, 147],
+            }
+        ]
+    },
+    average: {
+        labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+        series: [
+            {
+                name: "Week 1",
+                data: [150, 230, 224, 218, 135, 147, 260],
+                color: "#bbbbbb"
+            },
+            {
+                name: "Week 2",
+                data: [260, 218, 230, 224, 135, 150, 147],
+                color: "#bbbbbb"
+            },
+            {
+                name: "Week 3",
+                data: [230, 150, 218, 224, 147, 260, 135],
+                color: "#bbbbbb"
+            },
+            {
+                name: "Week 4",
+                data: [218, 224, 135, 260, 150, 230, 147],
+                color: "#bbbbbb"
+            },
+            {
+                name: "Average",
+                data: [214, 205, 201, 231, 141, 196, 172],
+                color: "#00aeff",
+                type: "line",
+                label: {
+                    show: true,
+                }
+            },
+        ]
+    },
+};
+
+@Component({
+    selector: 'hc-chart',
+    templateUrl: './chart-demo.component.html',
+    styleUrls: ['./chart-demo.component.scss']
+})
+
+export class ChartDemoComponent extends BaseDemoComponent {
+
+    @ViewChild('chartContainer') chartContainer:ElementRef;
+    chartTheme = 'light';
```

Comment:
After reading a little more the data binding to the input controls is not correct.  I think `@Input` is what I should have used.